dask/dataframe/groupby.py

```diff
@@ -293,6 +293,85 @@ def _var_agg(g, levels, ddof):
     return result
 
 
+def _cov_combine(g, levels):
+    return g
+
+
+def _cov_finalizer(df, cols):
+    vals = []
+    num_elements = len(list(it.product(cols, repeat=2)))
+    num_cols = len(cols)
+    vals = np.zeros(num_elements)
```

Comment:
The PR in it's current state doesn't support multicolumn groupby -- i will need rework `_cov_finalizer`